src/bin/main.rs

```diff
@@ -0,0 +1,22 @@
+#[macro_use]
+extern crate log;
+
+use actix_web::{App, HttpServer};
+
+use hilow::router::router;
+
+#[actix_web::main]
+async fn main() -> std::io::Result<()> {
+    if cfg!(debug_assertions) {
+        dotenv::from_filename(".env.development")
+            .expect("Unable to find \".env.development\" in the CWD");
+    }
+
+    env_logger::init();
+
+    info!("Serving on http://0.0.0.0:7878");
+    HttpServer::new(|| App::new().configure(router))
+        .bind("0.0.0.0:7878")?
+        .run()
+        .await
+}
```

Comment:
I suggest to keep `main.rs` as clean as possible, just resources initializations and Actix startup configuration allowed.

This would help us to have a clean project structure, I really want to encourage the use of a router in a separated module to avoid importing `handlers` and any domain related values to the HTTP server initialization.

What do you guys think?